awsx/ecs/ec2TaskDefinition.test.ts

```diff
@@ -45,14 +45,20 @@ describe("validateNetworkMode", () => {
 
   it("the default network mode should be awsvpc", async () => {
     const taskDefinition = new EC2TaskDefinition("default", {
-      container: {},
+      container: {
+        name: "fake-name",
+        image: "fake-image",
+      },
```

Comment:
Do users have to add these properties? Won't old code work without doing so? Does it mean this change is breaking?